README.md

```diff
@@ -423,7 +423,7 @@ In addition, uv respects the following environment variables:
 
 uv supports custom CA certificates (such as those needed by corporate proxies) by utilizing the
 system's trust store. To ensure this works out of the box, ensure your certificates are added to the
-system's trust store.
+system's trust store, and run uv with the `--native-tls` command-line Flag.
```

Comment:
I guess I would rephrase since this isn't quite "out of the box" anymore?

Reply:
I'd say... "By default, we use Mozilla's...." as you do in the docstring as well